data/json/bionics.json

```diff
@@ -648,7 +648,7 @@
     "id": "bio_night_vision",
     "type": "bionic",
     "name": "Implanted Night Vision",
-    "description": "Your eyes have been surgically modified to amplify existing light, allowing you to see in the dark.",
+    "description": "Your eyes have been surgically modified to amplify existing light, allowing you to see in the dark at the cost of power.",
```

Comment:
What about something like "When active, this bionic implanted into your eyes, amplify existing light. Allowing you to see in the dark." ? To keep the same kind of formulation than other power draining CBM.